framework/src/org/apache/cordova/engine/SystemWebChromeClient.java

```diff
@@ -199,7 +198,6 @@ public void onHideCustomView() {
      * @return View The progress view.
      */
     public View getVideoLoadingProgressView() {
-
```

Comment:
ditto on cleanup (...)

Reply:
Extracted the whitespace changes to a separate commit and updated the PR description.